src/test/resources/feature/3-destinatario/personaGiuridica/deleghe/018_001_delegatoPagaNotificaPG.feature

```diff
@@ -0,0 +1,45 @@
+Feature:Il delegato persona giuridica accede ad una delega
+
+  @TestSuite
+  @TA_PGdelegatoPagaNotifica
+  @DeleghePG
+  @PG
+
+  Scenario: PN-10389 - Il delegato persona giuridica paga una notifica
+    Given PG - Si effettua la login tramite token exchange come "delegante", e viene visualizzata la dashboard
+    When Si inizializzano i dati per la notifica
+      | modello         | A/R                |
+      | documenti       | 1                  |
+      | oggettoNotifica | Pagamento rata IMU |
+      | costiNotifica   | false              |
+    And Si aggiunge un destinatario alla notifica
+      | indirizzo         | VIA ROMA 20        |
+      | dettagliIndirizzo | Scala b            |
+      | codicePostale     | 20147              |
+      | comune            | Milano             |
+      | dettagliComune    | Milano             |
+      | provincia         | MI                 |
+      | stato             | Italia             |
+      | nomeCognome       | Convivio Spa |
+      | codiceFiscale     | 27957814470   |
+      | tipoDestinatario  | PG                 |
+      | domicilioDigitale | test@test.com      |
+      | avvisoPagoPa      | 1                  |
+      | F24               | 1                  |
+    Then Creo in background una notifica per destinatario tramite API REST
+    And Aspetta 10 secondi
```

Comment:
create another step which use only webtool.waittime

this step, use pause but reload page